packages/core/src/types.ts

```diff
@@ -22,7 +22,7 @@ export interface ILoggable<S extends LogSeverity = LogSeverity> {
   channel: LogChannelName;
   timestamp: number;
   message: string;
-  context: any[];
+  context: unknown[];
```

Comment:
`unknown` is a bit more strict than `any` as a type, and thus preferred.

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-0.html#:~:text=unknown%20is%20the%20type%2Dsafe,to%20a%20more%20specific%20type.